modules/sso.php

```diff
@@ -1016,32 +1042,27 @@ public function msg_login_by_jetpack( $message ) {
 		 */
 		$msg = apply_filters( 'jetpack_sso_disclaimer_message', $msg );
 
+		if ( empty( $msg ) ) {
+			return $message;
+		}
+
 		$message .= sprintf( '<p class="message">%s</p>', $msg );
 		return $message;
 	}
 
 	/**
-	 * Error message displayed on the login form when the user attempts
-	 * to post to the login form and it is disabled.
+	 * Message displayed when the user can not be found after approving the SSO process on WordPress.com
 	 *
-	 * @since 2.8
 	 * @param string $message
-	 * @param string
-	 **/
-	public function error_msg_login_method_not_allowed( $message ) {
-		$err = __( 'Login method not allowed' , 'jetpack' );
-		$message .= sprintf( '<p class="message" id="login_error">%s</p>', $err );
-
-		return $message;
-	}
+	 * @return string
+	 */
 	function cant_find_user( $message ) {
-		if ( self::match_by_email() ) {
-			$err_format = __( 'We couldn\'t find an account with the email <strong><code>%1$s</code></strong> to log you in with.  If you already have an account on <strong>%2$s</strong>, please make sure that <strong><code>%1$s</code></strong> is configured as the email address, or that you have connected to WordPress.com on your profile page.', 'jetpack' );
-		} else {
-			$err_format = __( 'We couldn\'t find any account on <strong>%2$s</strong> that is linked to your WordPress.com account to log you in with.  If you already have an account on <strong>%2$s</strong>, please make sure that you have connected to WordPress.com on your profile page.', 'jetpack' );
-		}
-		$err = sprintf( $err_format, $this->user_data->email, get_bloginfo( 'name' ) );
-		$message .= sprintf( '<p class="message" id="login_error">%s</p>', $err );
+		$error = esc_html__(
+			"We couldn't find an account to log you in with. If you already have an account, please make sure that you have connected it to WordPress.com.",
```

Comment:
We couldn't find your account. If you already have an account, make sure you have connected to WordPress.com.